lua/kubectl/actions/commands.lua

```diff
@@ -23,7 +23,7 @@ function M.configure_command(cmd, envs, args)
     if state.context["current-context"] then
       table.insert(result.args, "--context")
       table.insert(result.args, state.context["current-context"])
-      vim.print(result.args)
+      -- vim.print(result.args)
```

Comment:
Figured this was accidentally left behind as a debugging measure.

Reply:
Thank you I'll remove it in main right away 🙏